app/src/main/java/com/example/reflector_android/Models/BlogRecyclerAdapter.kt

```diff
@@ -30,14 +30,14 @@ class BlogRecyclerAdapter(val articles: MutableList<Article>?) : RecyclerView.Ad
         when (article?.author) {
             null -> {
                 holder?.itemView.progressBar.visibility = View.VISIBLE
-                holder?.itemView.textView_Article.text = ""
+                holder?.itemView.Description.text = ""
                 holder?.itemView.Title.text = ""
                 holder?.itemView.Author.text = ""
```

Comment:
We should make all of the ids for these views into camelCase like you do for `progressBar` and `pubDate`